lib/application_insights/rack/track_request.rb

```diff
@@ -24,6 +24,9 @@ def initialize(app, instrumentation_key, buffer_size = 500, send_interval = 60)
       # Track requests and send data to Application Insights asynchronously.
       # @param [Hash] env the rack environment.
       def call(env)
+        id = rand(16**32).to_s(16)
```

Comment:
I need to learn some basics of Rack. To really make correlation work, you also want to handle the RequestId of the incoming request and append some guid, but I don't see where it is handle.

Reply:
I'll update support this.

My original idea would be to allow the user to update the env variable if they wished - doing it automatically seems better though :+1: